go2eiko.py

```diff
@@ -0,0 +1,439 @@
+                    if callable(name_converter):
+                        filename = name_converter(old_filename)
+                    elif isinstance(name_converter, str) and name_converter == "index.ts":
+                        filename = intex_dot_ts2name_converter(index_dot_ts, old_filename)
+                    with open(os.path.join(MASTER_OUTPUT_PATH, output_path, folder.lower(), filename), 'wb') as f:
+                        f.write(response.content)
+                    logging.info(f"Downloaded {filename}!")
+
+
+def download_artifacts():
+
+    # Defining how to extract the names of the files
+    def name_converter(old_filename: str) -> str:
+
+        # Define the pattern
+        pattern = r"UI_RelicIcon_\d+_(\d)\.([a-z]+)"
+
+        # Get extension
+        ext = re.search(pattern, old_filename).group(2)
+
+        # Get the filename
+        number2artifact = {
+            "1": "coupe",
+            "2": "plume",
+            "3": "couronne",
+            "4": "fleur",
+            "5": "sablier",
+        }
+        filename = number2artifact[re.search(pattern, old_filename).group(1)]
+        return f"{filename}.{ext}"
+
+    # Downloading the artifacts
+    download_recursively(
+        path="libs/gi/assets/src/gen/artifacts",
+        output_path="gamedata/assets/artifacts/",
+        name_converter=name_converter,
+        force_download=FORCE,
+        expected_files=5,
+    )
+
+
+def download_characters():
+    
+    # Character's base assets
+    download_recursively(
+        path = "libs/gi/assets/src/gen/chars",
+        output_path="gamedata/assets/characters/",
+        name_converter="index.ts",
+        force_download=FORCE,
+        expected_files=15,
+    )
+    
+    # Character namecards
+    def name_converter(old_filename: str) -> str:
+        ext = old_filename.split(".")[-1]
+        return f"card.{ext}"
+    
+    def output_path(old_filename: str) -> str:
+        if "Character" in old_filename:
+            pattern = "Character_([A-Za-z_]+)_Card\.[a-z]+"
+            character_name = re.search(pattern, old_filename).group(1).lower()
+        elif "Traveler" in old_filename:
+            if "Traveler_Female" in old_filename:
+                character_name = "travelerf"
+            elif "Traveler_M" in old_filename:
+                character_name = "travelerm"
+        while "_" in character_name:
+            character_name = character_name.replace("_", "")
+        return os.path.join("gamedata/assets/characters", character_name)
+    
+    download_folder(
+        path="libs/gi/char-cards/src",
+        output_path=output_path,
+        name_converter=name_converter,
+    )
+    
+    
+
+
+def download_weapons():
+
+    def name_converter(old_filename: str) -> str:
+        
+        # Get the extension
+        ext = old_filename.split(".")[-1]
+
+        # Check in which case we are
+        if "Awaken" in old_filename:
+            return f"weapon-awaken.{ext}"
```

Comment:
eikonomiya uses `weapon-awake`, not `weapon-awaken`

Reply:
Fixed in ed9428c16dbeea8794ec887d62dab078968c52fe